ext/node/polyfills/internal/blocklist.mjs

```diff
@@ -0,0 +1,218 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+
+import { primordials } from "ext:core/mod.js";
+import {
+  op_blocklist_add_address,
+  op_blocklist_add_range,
+  op_blocklist_add_subnet,
+  op_blocklist_check,
+  op_blocklist_new,
+  op_socket_address_parse,
+} from "ext:core/ops";
+
+import {
+  validateInt32,
+  validateObject,
+  validatePort,
+  validateString,
+  validateUint32,
+} from "ext:deno_node/internal/validators.mjs";
+import { ERR_INVALID_ARG_VALUE } from "ext:deno_node/internal/errors.ts";
+import { customInspectSymbol } from "ext:deno_node/internal/util.mjs";
+import { inspect } from "ext:deno_node/internal/util/inspect.mjs";
+
+const {
+  Boolean,
+  Symbol,
+} = primordials;
+
+const kRid = Symbol("resourceId");
+
+class BlockList {
+  constructor() {
+    this[kRid] = op_blocklist_new();
```

Comment:
There is no way to close this resource, right?

That is quite problematic, because if you can't close it, you can't call anything that uses this in a `Deno.test`. @littledivy is cppgc integration ready to use here?

Reply:
Yes, this can be written with cppgc objects

Reply:
Thanks for instructions on cppgc, Divy. The resource leak is fixed.